pippo-core/src/main/java/ro/pippo/core/route/ClasspathResourceHandler.java

```diff
@@ -32,10 +38,20 @@ public ClasspathResourceHandler(String urlPath, String resourceBasePath) {
         this.resourceBasePath = getNormalizedPath(resourceBasePath);
     }
 
+    private static Path ROOT_PATH = Paths.get("/");
+    private static String normalizePath(String subPath) {
+        return ROOT_PATH.relativize(ROOT_PATH.resolve(subPath).normalize()).toString();
+    }
+
     @Override
     public URL getResourceUrl(String resourcePath) {
-        String resourceName = getResourceBasePath() + "/" + resourcePath;
-        return this.getClass().getClassLoader().getResource(resourceName);
+        String normalizedPath = normalizePath(resourcePath);
```

Comment:
Sure, but I wont touch the `WebjarsResourceHandler`, I don't understand how it should work and don't really have the time to dive in. Feel free to change anything or not use this pull request.